pkg/cmd/logs.go

```diff
@@ -382,9 +385,15 @@ func logsControllerManager() {
 }
 
 // logsVpnSeed prints the logfile of the vpn-seed container
-func logsVpnSeed(shootName string) {
+func logsVpnSeed(shootTechnicalID string) {
 	fmt.Println("-----------------------Kube-Apiserver")
-	logPodSeed("kube-apiserver", shootName, "vpn-seed")
+	if shootTechnicalID == emptyString {
+		output, err := ExecCmdReturnOutput("bash", "-c", "kubectl config current-context")
```

Comment:
Here if the user already has a context defined we should not need to run the kubectl command again

Reply:
It's actually through the command that I'm getting the context. Is there another way to get it ?

Reply:
ah i see! Nvm my previous message in that case 😅 

I would just say to invoke `kubectl` directly without relying bash (something like `ExecCmdReturnOutput("kubectl", "config", "current-context"`)

Reply:
Done